### PR TITLE
New generic class for commands they are changig prompt

### PR DESCRIPTION
**Pull request by marcin-usielski**
New generic class for commands they are changig prompt

### CODE REVIEW

File: moler/cmd/commandchangingprompt.py [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+        """
+        super(CommandChangingPrompt, self).__init__(connection=connection, prompt=prompt, newline_chars=newline_chars,
+                                                    runner=runner)
+
+        # Parameters defined by calling the command
+        self._re_expected_prompt = CommandTextualGeneric._calculate_prompt(expected_prompt)  # Expected prompt on device
+        self._re_prompt_after_login = self._re_expected_prompt
+        if prompt_after_login:
+            self._re_prompt_after_login = CommandTextualGeneric._calculate_prompt(prompt_after_login)
+        self._re_failure_exceptions_indication = None
```
Comment (Ernold11) [inline moler/cmd/commandchangingprompt.py]: move down to internal variable section

File: moler/cmd/commandchangingprompt.py [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+
+from moler.cmd.commandtextualgeneric import CommandTextualGeneric
+from moler.exceptions import ParsingDone
+
+
+@six.add_metaclass(abc.ABCMeta)
+class CommandChangingPrompt(CommandTextualGeneric):
+    """Base class for textual commands to change prompt."""
+
+    def __init__(self, connection, newline_chars=None, prompt=None, runner=None, expected_prompt=r'^>\s*',
```
Comment (Ernold11) [inline moler/cmd/commandchangingprompt.py]: command will change prompt so in my opinion expected_prompt should not have default value

Comment (marcin-usielski) [reply moler/cmd/commandchangingprompt.py]: Done

File: moler/cmd/commandchangingprompt.py (hunk 1 above)
Comment (marcin-usielski) [reply moler/cmd/commandchangingprompt.py]: Done

**Pull request merged**